examples/user_guide/Linking_Plots.ipynb

```diff
@@ -0,0 +1,220 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from holoviews.plotting.links import DataLink\n",
+    "\n",
+    "scatter1 = hv.Scatter(np.random.randn(1000, 2))\n",
+    "scatter2 = hv.Scatter(np.random.randn(1000, 2)*2, 'x2', 'y2')\n",
+    "\n",
+    "DataLink(scatter1, scatter2)\n",
+    "\n",
+    "(scatter1 + scatter2).options('Scatter', tools=['box_select'], clone=False)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Writing a Link\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "A ``Link`` consists of two components the ``Link`` itself and a ``LinkCallback`` which provides the actual implementation behind the ``Link``. In order to demonstrate writing a ``Link`` we'll start with a fairly straightforward example, linking an ``HLine`` to the mean y-value of a selection on a ``Scatter`` element. To express this we declare a ``MeanHLineLink`` class subclassing from the ``Link`` baseclass and declare ``ClassSelector`` parameters with the appropriate types to perform some basic validation."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import param\n",
+    "from holoviews.plotting.links import Link\n",
+    "\n",
+    "class MeanHLineLink(Link):\n",
+    "    \n",
+    "    source = param.ClassSelector(class_=hv.Scatter)\n",
+    "    target = param.ClassSelector(class_=hv.HLine)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we have the ``Link`` class we need to write the implementation in the form of a ``LinkCallback``. A ``LinkCallback`` should declare the ``source_model`` we want to listen to events on and a ``target_model``, declaring which model should be altered in response. To find out which models we can attach the ``Link`` to we can create a ``Plot`` instance and look at the ``plot.handles``, e.g. here we create a ``ScatterPlot`` and can see it has a 'source', which represents the ``ColumnDataSource``."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "renderer = hv.renderer('bokeh')\n",
+    "\n",
+    "plot = renderer.get_plot(hv.Scatter([]))\n",
+    "\n",
+    "plot.handles.keys()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this case we are interested in the 'source' handle, but we still have to tell it which events should trigger the callback. Bokeh callbacks can be grouped into two types model property changes and and events. For more detail on these two types of callbacks see the [Bokeh user guide](https://bokeh.pydata.org/en/latest/docs/user_guide/interaction/callbacks.html#userguide-interaction-jscallbacks).\n",
+    "\n",
```

Comment:
'into two types model property changes'? I suspect some punctuation is missing here...

Reply:
Thanks.